docs/999-big-list-of-options.md

```diff
@@ -1053,15 +1053,15 @@ Type: `((id: string) => string) | { [id: string]: string }`<br>
 Same as [`context`](guide/en/#context), but per-module – can either be an object of `id: context` pairs, or an `id => context` function.
 
 #### output.amd
-Type: `{ id?: string, define?: string}`
+Type: `{ id?: string, autoId?: boolean, basePath?: string, define?: string }`
```

Comment:
While I would keep the type in the documentation as written here and not use the complicated type I suggested for `types.d.ts`, this section should mention that `id` can only be used for single-file builds and cannot be combined with `autoId` or `basePath`.